src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf

```diff
@@ -140,15 +140,15 @@
             </trans-unit>
             <trans-unit id="38">
                 <source>This value is not a valid language.</source>
-                <target>Deze waarde representeert geen geldige taal.</target>
+                <target>Deze waarde is geen geldige taal.</target>
             </trans-unit>
             <trans-unit id="39">
                 <source>This value is not a valid locale.</source>
-                <target>Deze waarde representeert geen geldige lokalisering.</target>
+                <target>Deze waarde is geen geldige locale.</target>
```

Comment:
Hmz i get correct results on google.nl, also vice versa.
![image](https://cloud.githubusercontent.com/assets/1047696/17480784/e4b052fe-5d79-11e6-98c3-8b9fd9a8679d.png)

`local` is something different (basicly refers something that belongs to a certain place, ie "the local food was great").

Anyway, `locale` definitely is the _right_ translation here.